frontend/src/metabase/querying/components/FilterModal/FilterTitle/FilterTitle.tsx

```diff
@@ -6,8 +6,8 @@ import type * as Lib from "metabase-lib";
 
 import { FilterColumnName } from "../FilterColumnName";
 
-import { InfoIcon, HoverParent } from "./FilterTitle.styled";
-export { HoverParent };
+import { InfoIcon } from "./FilterTitle.styled";
+export { HoverParent } from "metabase/components/MetadataInfo/ColumnInfoIcon";
```

Comment:
This violates `import/order` rule :thinking:

Reply:
Strange, I would've expected it to come up in the linter.

Reply:
Oh, it's an export, I didn't see that before.

Why are we exporting it from here?

Reply:
It's coupled to the `InfoTitle` component (in the file it's being exported from), so I feel like the exports should be kept together.